plugins/@grouparoo/pipedrive/src/lib/parallelism.ts

```diff
@@ -0,0 +1,24 @@
+import { AppParallelismMethod, objectCache } from "@grouparoo/core";
+
+const DEFAULT_VALUE = 10;
```

Comment:
Let's make this `7` here. Then we will fan out to more once we learn the plan they are on.
This is the lowest plan (20) divided by 3.